Sitefinity CLI/Commands/AddModuleCommand.cs

```diff
@@ -13,13 +13,6 @@ namespace Sitefinity_CLI.Commands
     [Command(Name = Constants.AddModuleCommandName, Description = "Adds a new module to the current project.", FullName = Constants.AddModuleCommandFullName)]
     internal class AddModuleCommand : AddToSolutionCommandBase
     {
-        /// <summary>
-        /// The description of the module, visible in Sitefinity / Modules and Services
-        /// </summary>
-        [Option(Constants.DescriptionOptionTemplate, Constants.DescriptionOptionDescription, CommandOptionType.SingleValue)]
-        [DefaultValue("")]
-        public string Description { get; set; }
-
```

Comment:
I thought that we will just remove description from the config file and therefore from the VSIX but not from the CLI. Is that what you agreed on with Vladi?

Reply:
We agreed the Description in the templates to be empty string along with an inline comment.